docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/obstructed_labor.rst

```diff
@@ -161,8 +161,8 @@ while dashed lines indicate pieces of the underlying pregnancy model.
         rankdir = LR;
         ftp [label="full term\npregnancy, post\nintrapartum", style=dashed]
         ftb [label="full term\nbirth", style=dashed]
-        alive [label="parent alive"]
-        dead [label="parent dead"]
+        alive [label="parent did not die of cause"]
+        dead [label="parent died of cause"]
```

Comment:
Instead of being generic and saying "cause", I think it would be better to be specific and say "obstructed labor" or "OL". Might be worth changing the variable names of the "alive" and "dead" nodes as well, for clarity, though I'm not sure what would be clear, short names.